html2docx.py

```diff
@@ -1,155 +1,183 @@
-import io
-import re
+"""Parser for converting HTML to DOCX
+
+Accept input from a source like TinyMCE generated HTML. The
+source HTML may be preprocessed to limit tags and attributes.
+The input HTML must be valid with all closing tags (except
+for `br` tag).
+
+This module assumes a few conventions:
+
+- Block elements: `h1` to `h6`, `p`, `ol`, `ul`.
+- Any text inside a block element (`div`, `blockquote`, `pre`) that is
+  not inside a `p` element will have its own `p` element created
+  for it. The block element itself is not represented in the
+  resulting document.
+"""
+
 from html.parser import HTMLParser
-from typing import List, Tuple
+from io import BytesIO
+from typing import Dict, List, Tuple
+import re
 
 from docx import Document
-from docx.text.paragraph import Paragraph
+from docx.enum.text import WD_ALIGN_PARAGRAPH
+from docx.shared import Pt
+
+
+PT_WHITESPACE = re.compile(r"\s+")
```

Comment:
Why was this renamed form `WHITESPACE_RE`? Unless there is a reason, please follow the established convention.